lib/helper.js

```diff
@@ -0,0 +1,12 @@
+"use strict";
+
+const SparqlConverter = require("./sparql_converter");
+
+module.exports = {
+    program_to_sparql(program){
+        return SparqlConverter.program_to_sparql(program);
+    },
+    table_to_sparql(table){
+        return SparqlConverter.table_to_sparql(table);
+    },
+};
```

Comment:
I think it would be better to merge these two with one "toSparql" exposed to users. You can determine the type by `isTable` and `isProgram`. Note that, programs can potentially have no table in it. Although in practice we won't do that, but we should avoid using things like `program["rules"][0]["table"]["table"]` without any protection